data/Music/Sonic the Hedgehog Series/Sonic 2 (8 bit)/Green Hills Zone.meta

```diff
@@ -0,0 +1,4 @@
+Sonic the Hedgehog 2 (8 bit) - Green Hills Zone
+0x03
+bgm
+HyruleField,Fields
```

Comment:
The location line should have at least one group from each level of specificity (exact, high, mid and low). We can merge the PR without this, but then someone else will come in and add the missing groups for you, just so you know.